docs/Mock-Core-Configs.md

```diff
@@ -54,7 +54,7 @@ distribution.
 | [Fedora](https://fedoraproject.org/)                                           | `fedora-*`        | [Mock team][]                                                         | [Issues](https://github.com/rpm-software-management/mock/issues) |
 | [Kylin](https://kylinos.cn/)                                                   | `kylin-*`         | [@scaronni](https://github.com/scaronni)                              | NA  |
 | [Mageia](https://www.mageia.org/en/)                                           | `mageia-*`        | [@Conan-Kudo](https://github.com/Conan-Kudo)                          | [Issues](https://bugs.mageia.org/)  |
-| [Navy Linux](https://navylinux.org/)                                           | `navy-*`          | N/A                                                                   | N/A |
+| [Navy Linux](https://navylinux.org/)                                           | `navy-*`          | [@navylinux](https://github.com/navylinux)                             | [Issues](https://github.com/navy-linux/issue-tracker/issues) |
```

Comment:
Would you mind mentioning either GH team (which is a list of people, see the Mock team exmaple), or mentioning yourself directly?  The thing is that we don't know who can we CC in case of any issues.